app/locustio/jira/http_actions.py

```diff
@@ -106,6 +108,8 @@ def create_issue_open_quick_create():
                                   'fields_to_retain': fields_to_retain,
                                   'custom_fields_to_retain': custom_fields_to_retain
                                   }
+        if not ('"id":"project","label":"Project"' in content):
+            logger.error(params.err_message_create_issue)
```

Comment:
need to add content like:
logger.error(f'User {user[0]} authentication failed: {content}')